app/questionPage.html

```diff
@@ -45,7 +45,12 @@ <h3 id="timer"> 30m 00s </h3>
                 <button type="button" class="btn btn-info prev">Previous</button>
             </div>
             <div class="question-container col-md-8">
-                <div id="questions"></div>
+                <div id="questions">
+
```

Comment:
Please remove whitespace so lines 48 and 50 are on one line

Reply:
resolved in 645d1d4